source/features/highest-rated-comment.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import './highest-rated-comment.css';
+
+type Option = {
+	$el: Element;
+	likes: number;
+	unlikes: number;
+	index: number;
+}
+type Props = {
+	id: string;
+	username: string;
+	text: string;
+	avatar: string;
+}
+
+const like = '👍';
+const unlike = '👎';
+
+const element = ({id, username, text, avatar}: Props): Node => (
+	<div className="timeline-comment-wrapper rgh-highest-rated-comment">
+		<div className="avatar-parent-child timeline-comment-avatar">
+			<a href={`https://github.com/${username}`} className="d-inline-block">
+				<img src={avatar} alt={username} className="avatar rounded-1" height="44" width="44"/>
+			</a>
+		</div>
+
+		<a href={`#${id}`}>
+			<div className="bg-white border details-reset rounded-1">
+				<div className="bg-gray border-bottom-0 px-2 py-0 rgh-highest-rated-comment-summary">
+					<div className="d-flex flex-items-center">
+						<a href={`#${id}`} className="btn btn-sm rgh-highest-rated-comment-btn">
+							<svg height="16" className="octicon octicon-arrow-down" viewBox="0 0 10 16" version="1.1" width="20" aria-hidden="true">
+								<path fill-rule="evenodd" d="M7 7V3H3v4H0l5 6 5-6H7z"></path>
+							</svg>
```

Comment:
This icon should be part of `icons.tsx`. Also `viewBox` and `version` attributes should be dropped, and the other attributes sorted like the other icons are in there.